tests/src/App/ElementMap.cpp

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#include "gtest/gtest.h"
+
+#include "App/ElementMap.h"
+
+#include <sstream>
+
+// NOLINTBEGIN(readability-magic-numbers)
+
+class ElementMapTest : public ::testing::Test {
+};
+
+TEST_F(ElementMapTest, defaultConstruction)
+{
+  // Act
+
+  // Assert
+}
```

Comment:
Needs to close the `NOLINTBEGIN` (or just eliminate it, since there aren't any magic numbers here 😁 ).